src/ol/format/kml.js

```diff
@@ -1078,6 +1108,9 @@ ol.format.KML.readStyle_ = function(node, objectStack) {
   var imageStyle = /** @type {ol.style.Image} */
       ('imageStyle' in styleObject ?
           styleObject['imageStyle'] : ol.format.KML.DEFAULT_IMAGE_STYLE_);
+  if (imageStyle == ol.format.KML.DEFAULT_NO_IMAGE_STYLE_) {
```

Comment:
Very good; I'll make that modification next week.

On Sun, Aug 21, 2016 at 4:46 PM, Andreas Hocevar <notifications@github.com>
wrote:

> In src/ol/format/kml.js
> <https://github.com/openlayers/ol3/pull/5745#discussion_r75606099>:
>
> > @@ -1078,6 +1108,9 @@ ol.format.KML.readStyle_ = function(node, objectStack) {
> >    var imageStyle = /** @type {ol.style.Image} */
> >        ('imageStyle' in styleObject ?
> >            styleObject['imageStyle'] : ol.format.KML.DEFAULT_IMAGE_STYLE_);
> > +  if (imageStyle == ol.format.KML.DEFAULT_NO_IMAGE_STYLE_) {
>
> Thanks for the explanation. In this case I'd opt for null - smaller build
> size and the name of the constant has all information that's needed when
> reading the code.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openlayers/ol3/pull/5745/files/e0d75555c5a110b24fe63e618966ac73993c198c#r75606099>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFgK8A-2ZkXr5NtRdxbX1QqmmMDj6LUOks5qiONTgaJpZM4Jm_9w>
> .
>